src/linalg/basic/matrix.rs

```diff
@@ -304,6 +384,8 @@ where
 impl<T: Debug + Display + Copy + Sized> Array<T, (usize, usize)> for DenseMatrix<T> {
     fn get(&self, pos: (usize, usize)) -> &T {
         let (row, col) = pos;
+        let idx_target = col * self.nrows + row;
```

Comment:
it seems that `idx_target` is not used? maybe it was used for debugging purposes?